### ISSUE TITLE
Syntax error halting app from running

### ISSUE THREAD

**Issue by jdcrecur** [interfacewerk/ng2-stateful-button#5]
ERROR
`Uncaught SyntaxError: Unexpected token export`

CULPRIT
```
export * from './src/';

/*****************
 ** WEBPACK FOOTER
 ** ./~/ng2-stateful-button/index.js
 ** module id = 430
 ** module chunks = 0
 **/
```

I have just followed the guide in the read me, imported the module and included in the imports array.. have not actually used the module yet as is halting the app from loading.

**Comment by kevinmerckx** [comment]
Hi @johnc1984 , we will have a look at it. Thanks for reporting the issue!

**Comment by kevinmerckx** [comment]
@johnc1984 Could you tell me how you installed the module?

with

```
npm install ng2-stateful-button --save
```

or did you follow the Develop section of the README?

**Comment by jdcrecur** [comment]
I used from the installation section of the readme:
```
npm install ng2-stateful-button --save
```

This write into the package json:
```
    "ng2-stateful-button": "0.0.7",
```

Which resulted in the error. 

The basic app.module looks like:
```
// Core and Imports
import {NgModule}      from '@angular/core';
import {BrowserModule} from '@angular/platform-browser';
import {HttpModule} from '@angular/http';
import {FormsModule, ReactiveFormsModule} from '@angular/forms';
import {Router, RouterModule} from '@angular/router';
import {BrowserXhr, Http, XHRBackend, RequestOptions} from '@angular/http';

// code remove for brevity

//NgModule
@NgModule({
    imports: [
        BrowserModule,
        DndModule.forRoot(),
        HttpModule,
        FormsModule,
        ReactiveFormsModule,
        StatefulButtonModule,

```

We are building with webpack and webpack-dev server.


The code from npm and ithub actually appear to be completely different:
https://postimg.org/image/o1tkv2yaf/

**Comment by kevinmerckx** [comment]
We might have fixed the error that caused this issue: a typo! Your bundler (webpack) shouldn't take the file `index.js` but `bundles/ng2-stateful-button.umd.js` as specified in `package.json`.

Could you give it a try and tell us if it's fixed in version 0.0.8?
